src/odemis/driver/xt_client.py

```diff
@@ -1909,3 +1921,178 @@ def stop(self, axes=None):
+        beam_stigmator_range_y = beam_stigmator_info["range"]["y"]
+        beam_stigmator_range = tuple((i, j) for i, j in zip(beam_stigmator_range_x, beam_stigmator_range_y))
+        self.beamStigmator = model.TupleContinuous(
+            tuple(self.parent.get_primary_stigmator()),
+            unit=beam_stigmator_info["unit"],
+            range=beam_stigmator_range,
+            setter=self._setBeamStigmator)
+
+        pattern_stigmator_info = self.parent.secondary_stigmator_info()
+        pattern_stigmator_range_x = pattern_stigmator_info["range"]["x"]
+        pattern_stigmator_range_y = pattern_stigmator_info["range"]["y"]
+        pattern_stigmator_range = tuple((i, j) for i, j in zip(pattern_stigmator_range_x,
+                                                                   pattern_stigmator_range_y))
+        self.patternStigmator = model.TupleContinuous(
+            tuple(self.parent.get_secondary_stigmator()),
+            unit=pattern_stigmator_info["unit"],
+            range=pattern_stigmator_range,
+            setter=self._setPatternStigmator)
+
+        self.beamShiftTransformationMatrix = model.ListVA(
+            self.parent.get_dc_coils(),
+            unit=None,
+            readonly=True)
+
+        self.multiprobeRotation = model.FloatVA(
+            math.radians(self.parent.get_mpp_orientation()),
+            unit="rad",
+            readonly=True)
+
+        aperture_index_info = self.parent.aperture_index_info()
+        self.apertureIndex = model.IntContinuous(
+            int(self.parent.get_aperture_index()),
+            unit=aperture_index_info["unit"],
+            range=tuple(int(i) for i in aperture_index_info["range"]),
+            setter=self._setApertureIndex)
+
+        beamlet_index_info = self.parent.beamlet_index_info()
+        beamlet_index_range_x = beamlet_index_info["range"]["x"]
+        beamlet_index_range_y = beamlet_index_info["range"]["y"]
+        beamlet_index_range = tuple((int(i), int(j)) for i, j in zip(beamlet_index_range_x, beamlet_index_range_y))
+        beamlet_index = self.parent.get_beamlet_index()
+        self.beamletIndex = model.TupleContinuous(
+            tuple(int(i) for i in beamlet_index),  # convert tuple values to integers.,
+            unit=beamlet_index_info["unit"],
+            range=beamlet_index_range,
+            setter=self._setBeamletIndex
+        )
+
+        multibeam_mode = (self.parent.get_use_case() == 'MultiBeamTile')
+        self.multiBeamMode = model.BooleanVA(
+                multibeam_mode,
+                setter=self._setMultiBeamMode
+        )
+
+        # TODO If _updateSettings is updated move this to the top of the __init__ function. For now the update thread
+        #  can only be started after the MB VA's are initialized.
+        # Instantiate the super scanner class with the update thread
+        super(MultiBeamScanner, self).__init__(name, role, parent, hfw_nomag, **kwargs)
```

Comment:
Now, I understand why it'd be really nice to have a list of functions to call for updating the settings.
It's fine as it is now though. It's a little odd, but instantiating the VAs before the rest shouldn't cause any issues.